tests/integration/kubernetes/gha-run.sh

```diff
@@ -135,6 +135,10 @@ function deploy_kata() {
 	# Enable 'default_vcpus' hypervisor annotation
 	yq write -i "${tools_dir}/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml" 'spec.template.spec.containers[0].env[6].value' "default_vcpus"
 
+	if [ -n "${SNAPSHOTTER}" ]; then
+		yq write -i "${tools_dir}/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml" 'spec.template.spec.containers[0].env[7].value' "${KATA_HYPERVISOR}:${SNAPSHOTTER}"
+	fi
```

Comment:
It would be clearer of this was folded imho:

```suggestion
	if [ -n "${SNAPSHOTTER}" ]; then
		yq write -i \
		  "${tools_dir}/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml" \
		  'spec.template.spec.containers[0].env[7].value' \
		  "${KATA_HYPERVISOR}:${SNAPSHOTTER}"
	fi
```

Reply:
Fixed. And I refactor other code with yq for better clarity.